scripts/ci/create-plunker-bundle.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * This script creates a bundle of all components and publishes it to Firebase.
+ * The bundle will be used to load a Plunker Demo of Angular Material 2.
+ */
+
+const spawn = require('child_process').spawnSync;

```

Comment:
@hansl @jelbourn Addressed your comments. 

> Is there a way to only run this for master?

Travis is actually only running in `master`
> ```yml
branches:
  only:
  - master
```